api/api/clubs/apps.py

```diff
@@ -0,0 +1,7 @@
+from django.apps import AppConfig
+
+
+class ClubsConfig(AppConfig):
+    default_auto_field = 'django.db.models.BigAutoField'
+    name = 'api_clubs'
```

Comment:
name should be `api.clubs`
Look at the old apps for similarity.
For example: https://github.com/BitByte-TPC/gymkhana/blob/master/api/api/accounts/apps.py